lib/swagger/jsonLoader.ts

```diff
@@ -111,6 +111,65 @@ export class JsonLoader implements Loader<Json> {
     // throw new Error(`Unknown file format while loading file ${cache.filePath}`);
   }
 
+  private getFileContent(filePath: string, fileString: string): any {
+    if (
+      this.opts.supportYaml &&
```

Comment:
why yaml support is implemented in this pr?

Reply:
can be removed.